app/src/androidTest/java/com/sdpteam/connectout/profileList/CommunityActivityTest.java

```diff
@@ -90,10 +97,11 @@ public void clickingOnAUserLaunchesRightProfilePage() {
         Profile p = list.get(userIndexToCheck);
 
         String expectedUid = (p == null) ? NULL_USER : list.get(userIndexToCheck).getId();
+        System.out.println(expectedUid);
 
-/*TODO fix ProfileList bug
-        onView(withIndex(withId(R.id.nameAdapterTextView), userIndexToCheck)).perform(click());
-        intended(Matchers.allOf(hasComponent(ProfileActivity.class.getName()), hasExtra(equalTo("uid"), equalTo(expectedUid))));*/
+        //TODO fix ProfileList bug
+//        onView(withIndex(withId(R.id.nameAdapterTextView), userIndexToCheck)).perform(click());
+//        intended(Matchers.allOf(hasComponent(ProfileActivity.class.getName()), hasExtra(equalTo("uid"), equalTo(expectedUid))));
```

Comment:
I believe this TODO is one that is treated through this task, why does it remain here ?